libs/sdk-ui-filters/src/DateFilter/DateRangePicker/DateRangePicker.tsx

```diff
@@ -22,7 +22,7 @@ interface IDateRangePickerProps {
     range: IDateRange;
     onRangeChange: (newRange: IDateRange) => void;
     errors?: IExtendedDateFilterErrors["absoluteForm"];
-    dateFormat?: string | string[];
+    dateFormat?: string;
```

Comment:
Is it safe if we remove string[]?

Reply:
`DateRangePicker` is used only in `libs/sdk-ui-filters/src/DateFilter/AbsoluteDateFilterForm/AbsoluteDateFilterForm.tsx`.
It will be passed into `react-day-picker/DayPickerInput`, and we are not passing string array into `DayPickerInput`.
So it's safe

Reply:
ok